src/idd/idd.base.js

```diff
@@ -2885,9 +2911,7 @@ var _initializeInteractiveDataDisplay = function () { // determines settings dep
         // width, height are optional size of the element in the data space
         // returns added DOM element
         this.add = function (element, scaleMode, x, y, width, height, originX, originY) {
-            var el = $(element)
-            if(!this.host[0].contains(element))
```

Comment:
Please explain why you removed the 'contains' check which is required for Knockout-base initialization of IDD.